index.html

```diff
@@ -9,44 +9,216 @@
+        <img src="project-img.png" alt="project">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
+      <!-- card -->
+      <div class="card">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="portfolio-section2">
+    <div class="about-section">
+      <h1 class="about-title">About me</h1>
+      <p class="about-par"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hesitate to contact me.</p>
+      <button class="btn1">Get my resume</button>
+    </div>
+
+    <div class="all-about">
+<!-- card -->
+<div class="card about-card">
+  <img class="about-img" src="img3.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">languages</h2>
+    <ul class="skills">
+      <li class="skill skill-about">Ruby on Rails</li>
+      <li class="skill skill-about">Css</li>
+      <li class="skill skill-about">html</li>
+      <li class="skill skill-about">Javascript</li>
+    </ul>
+    <button class="btn btn-skill">See Project</button>
+  </div>
+</div>
+ <!-- card -->
+ <div class="card about-card">
+  <img class="about-img" src="img1.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">Farmwork</h2>
+    <ul class="skills">
+      <li class="skill skill-about">Ruby on Rails</li>
+      <li class="skill skill-about">Css</li>
+      <li class="skill skill-about">html</li>
+      <li class="skill skill-about">Javascript</li>
+    </ul>
+    <button class="btn btn-skill">See Project</button>
+  </div>
+</div>
+ <!-- card -->
+<div class="card about-card">
+  <img class="about-img" src="img2.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">Skills</h2>
+    <ul class="skills">
+      <li class="skill skill-about">Database Management</li>
+      <li class="skill skill-about">Version Control</li>
+      <li class="skill skill-about">CLI</li>
+      <li class="skill skill-about">Web Development</li>
+      <li class="skill ">API Design</li>
+      
+    </ul>
+    <button class="btn btn-skill">See Project</button>
+  </div>
+</div>
```

Comment:
Kindly fix this card styling to match the design

![image (21)](https://github.com/RolandAmini/portfolio-mobile/assets/59707859/61fadc50-b132-4071-add3-187963648f5c)